packages/components/pie-toast/src/index.ts

```diff
@@ -39,6 +51,13 @@ export class PieToast extends RtlMixin(LitElement) implements ToastProps {
     @property({ type: Boolean })
     public isOpen = defaultProps.isOpen;
 
+    @property()
+    @validPropertyValues(componentSelector, variants, defaultProps.variant)
+    public variant: NonNullable<ToastProps['variant']> = defaultProps.variant;
```

Comment:
I _think_ if you let TS infer the type from `defaultProps`, it will be required?
```suggestion
    public variant = defaultProps.variant;
```